deny.toml

```diff
@@ -21,6 +21,7 @@ allow = [
     "OFL-1.1",
     "Ubuntu-font-1.0",
     "CDLA-Permissive-2.0",
+    "bzip2-1.0.6"
```

Comment:
nit: We usually add the trailing comma, so a future addition is only a one-line nit, not a two-line one. (fixed, thanks!)